Gems/WhiteBox/Code/Source/Rendering/WhiteBoxRenderMeshInterface.h

```diff
@@ -41,8 +41,5 @@ namespace WhiteBox
 
         // Return if the White Box mesh is visible or not.
         virtual bool IsVisible() const = 0;
-
-        //! Set the White Box mesh visible (true) or invisible (false).
-        virtual void SetVisiblity(bool visibility) = 0;
```

Comment:
question: By removing this is it still possible to control the visibility separate from entity visibility using the material itself? I just want to ensure we're not removing useful functionality

Reply:
was wondering why even delegate visibility like this when in most cases all the mesh information is wiped from whitebox. you can see that is WhiteBoxEditorComponent.cpp